packages/ERTP/src/paymentLedger.js

```diff
@@ -400,39 +416,44 @@ export const makePaymentLedger = (
     withdrawInternal,
   };
 
-  const makeEmptyPurse = makePurseMaker(
+  const makeEmptyPurse = defineDurablePurse(
+    issuerBaggage,
     allegedName,
     assetKind,
     brand,
     purseMethods,
   );
 
-  /** @type {Issuer<K>} */
-  const issuer = Far(`${allegedName} issuer`, {
-    isLive,
-    getAmountOf,
-    burn,
-    claim,
-    combine,
-    split,
-    splitMany,
-    getBrand: () => brand,
-    getAllegedName: () => allegedName,
-    getAssetKind: () => assetKind,
-    getDisplayInfo: () => displayInfo,
-    makeEmptyPurse,
-  });
-
-  /** @type {Mint<K>} */
-  const mint = Far(`${allegedName} mint`, {
-    getIssuer: () => issuer,
-    mintPayment,
-  });
+  const dropContext =
```

Comment:
I only use `dropContext` for those methods that do not need the context. In the current experiment, where we state closed over by behavior, that is most of them. Curious what the other side of that experiment looks like in general; but it will certainly use context a lot more.